### DISCUSSION TITLE
Contributor Register

### DISCUSSION THREAD

**Discussion by noahstoner** [Finalgoal231/finalGoal#36] [Announcements]
# Hello, everyone

Welcome to contribute to this project.
To complete register to this organization, please drop your github username and email here.

And join at discord server:
https://discord.gg/RPMN8Pzh

**Comment by noahstoner** [comment]
Username: shinevue
Email: shine401020@gmail.com

**Comment by popstar7** [comment]
Username: joellake992
Email: joel.lake.777@gmail.com

**Comment by basitali717** [comment]
Username: techietrend
Email: techietrendsetter@gmail.com

**Comment by creative2113** [comment]
username: creative2113
```
scottking2113@gmail.com
```

**Reply by popstar7** [reply]
Hi, @creative2113 
Please follow input format

**Reply by creative2113** [reply]
what do you mean?

**Reply by creative2113** [reply]
input format??

**Reply by popstar7** [reply]
Username: techietrend
Email: techietrendsetter@gmail.com

like this

**Reply by creative2113** [reply]
why?
this is better than that.
we can copy faster.

**Reply by creative2113** [reply]
username: creative2113

Email: scottking2113@gmail.com
